lib/imgix/path.rb

```diff
@@ -75,10 +75,14 @@ def method_missing(method, *args, &block)
 
     ALIASES.each do |from, to|
       define_method from do |*args|
+        warn "Warning `Path.#{from}' has been deprecated and " \
+             "will be removed in the next major version.\n"
         self.send(to, *args)
       end
 
       define_method "#{from}=" do |*args|
+        warn "Warning `Path.#{from}=' has been deprecated and " \
```

Comment:
Nitpick: We should have a `:` after `Warning` to match the message format from #74

Reply:
Oof! Yes! Ty!